development/image-url-helper/pkg/promote/promote.go

```diff
@@ -224,3 +243,22 @@ func saveToFile(path string, lines []string) error {
 	}
 	return nil
 }
+
+func ParseExcludes(excludesListFilename string) ([]string, error) {
```

Comment:
argument excludesListFilename is getting value directly from excludeslist cli argument. Cli argument has description it contains list of excluded images. I expect to pass image names to this cli argument, not path to file with list excluded files. Description of cli argument is missleading.

Reply:
I've updated the CLI flag description to `Path to the file containing a list of excluded images`